### ISSUE TITLE
Add a .php_cs to make FlintCI working

### ISSUE THREAD

**Issue by Soullivaneuh** [chamilo/chamilo-lms#2274]
Hi, and thanks for using FlintCI! :-)

Just an issue to tell you have to add a `.php_cs` configuration file to make it working with php-cs-fixer.

FlintCI will just run the command, and it's currently failing: https://flintci.io/repositories/1013/analyses/406

More info: http://cs.sensiolabs.org/#config-file

Regards.

**Comment by Soullivaneuh** [comment]
Your project is quite huge and is still failing on FlintCI. Hopefully, I might found a solution for it. The fix will be available soon.

**Comment by Soullivaneuh** [comment]
I have now an another issue while storing the diff result of the analysis for this project. I'm still investigating on it.

**Comment by jmontoyaa** [comment]
I added more "exclude paths" in our main branches 1.11.x and master (.php_cs_dist)

**Comment by Soullivaneuh** [comment]
@jmontoyaa Yeah but the timeout is not the issue anymore. The produced diff has a weird encoding making my database yelling. For now this is the only concerned repo. Still investigating.

**Comment by Soullivaneuh** [comment]
Well, I have some progressions.

The master branch seems to work, but not 1.11.x. I'll try to reproduce and fix the issue.

**Comment by jmontoyaa** [comment]
@Soullivaneuh  thanks! Now it's working!
